libpod/events/events.go

```diff
@@ -65,6 +65,15 @@ func (e *Event) ToJSONString() (string, error) {
 	return string(b), err
 }
 
+// RotateEvent creates a new event for when the events logfile
+// is rotated and returns the rotate status and time.
+func (e *Event) RotateEvent(status Status) Event {
+	return Event{
+		Status: status,
+		Time:   time.Now(),
+	}
+}
```

Comment:
Why do we need this function? The name is not related to the logic.
You could just use `NewEvent(Rotate)`

Reply:
Why do we require an event at all? Do programs outside of Podman need to know that a rotation occurred?